validator/src/admin_rpc_service.rs

```diff
@@ -693,6 +693,10 @@ impl AdminRpcImpl {
                 .cluster_info
                 .set_keypair(Arc::new(identity_keypair));
             warn!("Identity set to {}", post_init.cluster_info.id());
+            solana_metrics::datapoint_info!(
```

Comment:
I think that all the data should be passed as it is done during the initialization of the validator, otherwise a query with a condition on the cluster won't return these data at all.